compiler/parser/src/lexer.rs

```diff
@@ -1366,18 +1368,27 @@ mod tests {
         assert_eq!(vec!['b', '\\', '\n'], x);
     }
 
-    fn stok(s: &str) -> Tok {
+    fn stok(s: &str, prefix: Option<String>, triple_quoted: bool) -> Tok {
         Tok::String {
             value: s.to_owned(),
             kind: StringKind::Normal,
+            prefix,
```

Comment:
If you prefer not to repeat yourself (using, e.g, `'r'.to_string()` in function calls), consider trying something like this:

```suggestion
    fn stok<T: ToString>(s: &str, prefix: Option<T>, triple_quoted: bool) -> Tok {
        Tok::String {
            value: s.to_owned(),
            kind: StringKind::Normal,
            prefix: prefix.map(|x| x.to_string()),
```

Reply:
@fanninpm Thanks!